oozie-to-airflow/validate-all-workflows

```diff
@@ -14,6 +14,5 @@
 # limitations under the License.
 set -euo pipefail
 MY_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
-ALL_WORKFLOWS=$(find ${MY_DIR} -name 'workflow.xml')
 
-${MY_DIR}/validate-workflows ${ALL_WORKFLOWS}
+find "${MY_DIR}" -name 'workflow.xml' | xargs -0 "${MY_DIR}/validate-workflows"
```

Comment:
Is this enforced by the linter? I find the previous implementation more readable.

Reply:
Yes. It required. 
````
In validate-all-workflows line 19:
${MY_DIR}/validate-workflows ${ALL_WORKFLOWS}
^-------^ SC2086: Double quote to prevent globbing and word splitting.
                             ^--------------^ SC2086: Double quote to prevent globbing and word splitting.
````

Reply:
Ok